### PR TITLE
Update getting-started-icp.md

### PR DESCRIPTION
**Pull request by alongzhi**
fix the formatting issue of pv.yaml, because it reports "error parsing pv.yaml" while copying to use directly.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1038)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
@alongzhi Please would you sign the CLA then come back and add a comment to this PR, as requested by the Googlebot in a previous comment.

**Comment by alongzhi** [comment]
@googlebot I signed it!

Regards，
A Long

**Comment by alongzhi** [comment]
I signed it!

Regards，
A Long

**Comment by sarahmaddox** [comment]
Thanks!
/assign

File: content/docs/started/getting-started-icp.md [hunk 1]
```diff
@@ -71,47 +71,47 @@ NAME         STATUS    ROLES                      AGE       VERSION
     The PVs defination file (`pv.yaml`) is as following:
 
      ```yaml
```
Comment (sarahmaddox) [inline content/docs/started/getting-started-icp.md]: The YAML file doesn't render properly. See preview:
https://deploy-preview-1038--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-icp/

Comment (alongzhi) [reply content/docs/started/getting-started-icp.md]: Oh sorry, I see it just now, I'm checking...

File: content/docs/started/getting-started-icp.md [hunk 2]
```diff
@@ -69,51 +69,51 @@ NAME         STATUS    ROLES                      AGE       VERSION
...
-        accessModes:
-        - ReadWriteOnce
-        persistentVolumeReclaimPolicy: Retain
-        nfs:
-          path: ${NFS_SHARED_DIR}/pv3
-          server: ${NFS_SERVER_IP}
-     ```
-
+    
+    ```yaml
```
Comment (sarahmaddox) [inline content/docs/started/getting-started-icp.md]: Suggestion: Copy the indentation of the image policy file on line 45.

Comment (alongzhi) [reply content/docs/started/getting-started-icp.md]: It works in website now after I deleted all blanks before each line. :-)

**Comment by sarahmaddox** [comment]
Please fix the indentation so that the YAML file and subsequent info all fall under the relevant list item. See preview:
https://deploy-preview-1038--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-icp/

Suggestion: Copy the indentation of the image policy file on line 45.

**Comment by alongzhi** [comment]
@sarahmaddox Thanks for your suggestions! As you have seen, even if I copied from the image policy section(line 45), it will show back to the rendering problem which you saw the first time yesterday. That's why I deleted all blanks before each line to let pv yaml section show normally. (I'm not sure why it's looks good in preview section every time after modified, but it's broken in website!)

**Comment by alongzhi** [comment]
I'd like to close this PR as the reason above. Thanks for your effort. @sarahmaddox

**Pull request closed without merging**